### ISSUE TITLE
BAM/CRAM Cleanup

### ISSUE THREAD

**Issue by PedalheadPHX** [tgen/phoenix#323]
The removal of BAM files if CRAM was generated is checking the IGV_links folder.  The "find" should exclude the IGV folder

**Comment by bryce-turner** [comment]
I believe this is more of a jetstream centro issue, as that is where we do the search for BAMs and validate if there is a corresponding CRAM.

We could modify the following line 5:
`for BAM in $(find "${PROJECT_PATH}" -name *.bam); do`
to be something similar to:
```bash
for BAM in $(find "${PROJECT_PATH}" -name *.bam -not -path "${PROJECT_PATH}/igv_symbolic_links/*"); do
```

**Comment by PedalheadPHX** [comment]
Yes something like that should do the trick, I now forget where I noticed this error, I'm guessing in a log file as clearly it doesn't cause a failure
